package.json

```diff
@@ -247,6 +252,12 @@
         "category": "ONE"
       }
     ],
+    "keybindings": [
+      {
+          "command": "one.explorer.runSingleSelectedCfg",
+          "key": "ctrl+f5"
```

Comment:
What about using another key binding? In Visual studio, Ctrl+F5 is used as the meaning of execution. I think it would be more appropriate to use this key binding to run tvn on simulator or target.